mmv1/third_party/terraform/services/networkservices/resource_network_services_gateway_test.go

```diff
@@ -63,11 +63,112 @@ resource "google_network_services_gateway" "foobar" {
   description = "update description"
   labels      = {
     foo = "bar"
-  } 
+  }
 }
 `, gatewayName)
 }
 
+func TestAccNetworkServicesGateway_networkServicesGatewaySecureWebProxyWithoutAddresses(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckNetworkServicesGatewayDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNetworkServicesGateway_networkServicesGatewaySecureWebProxyWithoutAddresses(context, true),
+			},
+			{
+				ResourceName:            "google_network_services_gateway.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"name", "location", "delete_swg_autogen_router_on_destroy", "labels", "terraform_labels"},
+			},
+		},
+	})
+}
+
+func testAccNetworkServicesGateway_networkServicesGatewaySecureWebProxyWithoutAddresses(context map[string]interface{}, withAddresses bool) string {
```

Comment:
Sorry I may misread something, but it looks to me that this test case is still testing the scenario where `addresses` is specified. Is this intended?

Reply:
Thanks for the check! Yes you are right. My idea was to add an integration test for SWP that could be easily reused to test other fields but I mess up with the names a bit.